src/user-management/interfaces/authenticate-with-session-cookie.interface.ts

```diff
@@ -34,4 +36,6 @@ export type AuthenticateWithSessionCookieSuccessResponse = {
   organizationId?: string;
   role?: string;
   permissions?: string[];
+  user?: User;
```

Comment:
Is user guaranteed on successful auth, or no?

Reply:
Should do, can't think of a scenario with successfully authenticating doesn't get you the user object.